jiant/tasks/lib/mrqa_natural_questions.py

```diff
@@ -0,0 +1,94 @@
+import gzip
+import json
+from dataclasses import dataclass
+
+from jiant.shared.constants import PHASE
+from jiant.tasks.lib.templates.squad_style import core as squad_style_template
+from jiant.utils.python.io import read_jsonl
+
+
+@dataclass
+class Example(squad_style_template.Example):
+    def tokenize(self, tokenizer):
+        raise NotImplementedError("SQuaD is weird")
```

Comment:
Best to change this to a `RuntimeError` and/or add a more descriptive error
https://docs.python.org/3/library/exceptions.html#NotImplementedError

Reply:
Let's resolve this jointly for all affected SQuAD-style tasks in a followup. New issue: #1210